configs/run_dev_se.config

```diff
@@ -0,0 +1,32 @@
+/************************************************
+| CONFIGURATION FILE FOR NAO VIRAL MGS WORKFLOW |
+************************************************/
+
+params {
+    mode = "run_dev_se"
+
+    read_type = "single_end"
```

Comment:
Related to above, shouldn't need to pass in read_type theoretically.

Reply:
resolved